scan_org_sboms.py

```
@@ -6,203 +6,52 @@
+                if p:
+                    items.append(p)
+    targets = set(items)
+    target_names = set([p.rsplit("@", 1)[0] for p in targets if "@" in p])
+    return targets, target_names
+
+try:
+    TARGETS, TARGET_NAMES = load_targets(SUSPICIOUS_FILE)
+except FileNotFoundError:
+    print(f"Suspicious list file not found: {SUSPICIOUS_FILE}", file=sys.stderr)
+    sys.exit(2)
+except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)